public/weaver/why-weaver/index.html

```
@@ -255,11 +255,11 @@ <h2 class="font-serif text-3xl text-weaver-indigo mb-6">CI/CD Supervision</h2>
                             <p class="text-weaver-indigo/80 mb-6">
                                 AI shouldn't just run on your laptop. Weaver is designed to be the supervisory layer in your continuous integration pipeline.
                             </p>
-                            <a href="../docs/" class="text-sm font-bold text-weaver-indigo border-b border-weaver-indigo pb-1 hover:text-weaver-vermilion hover:border-weaver-vermilion transition-colors">
+                            <a href="/weaver/docs/" class="text-sm font-bold text-weaver-indigo border-b border-weaver-indigo pb-1 hover:text-weaver-vermilion hover:border-weaver-vermilion transition-colors">
                                 View CI Integration Docs ->
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)